### PR TITLE
Add POWER AT12.0 and clang compilers for C based off existing crossbuild-ng

### PR DESCRIPTION
**Pull request by grooverdan**
Using AT12.0 and clang in C language too since we have it compiled.

Thanks @mattgodbolt.

### CODE REVIEW

**Comment by pozdneev** [comment]
Super! A code in C compiles successfully!

However, the following code in C++ fails to compile with "power64le clang (trunk)" and "powerpc64 clang (trunk)":
```
#include <cmath>

using namespace std;

int square(int num) {
    return num * num;
}
```
(The `-x c++` does not help.)

The error message is
```
[powerpc64 clang (trunk) #1] fatal error: 'cmath' file not found
```

**Comment by pozdneev** [comment]
> ```
> [powerpc64 clang (trunk) #1] fatal error: 'cmath' file not found
> ```

This is a known issue: #523.

File: etc/config/c.amazon.properties [hunk 1]
```diff
@@ -179,9 +179,21 @@ group.ccross.groupName=Cross GCC
 
 ###############################
 # GCC for PPC
-group.cppc.compilers=cppcg48:cppc64leg630
-group.cppc.groupName=PowerPC GCC
+group.cppc.compilers=cppcg48:cppc64leg630:ppc64leg8:ppc64g8:ppc64clang:ppc64leclang
```
Comment (RabsRincon) [inline etc/config/c.amazon.properties]: This IDs are going to need to change for this to be mergable. Could you please add a `c` before the ids (As done for `cppcg48` for example). Thanks! :)

Comment (grooverdan) [reply etc/config/c.amazon.properties]: done. sorry I missed that.

**Comment by grooverdan** [comment]
Thanks @partouf

**Pull request merged**